Sources/Plot/API/HTMLComponents.swift

```diff
@@ -579,6 +579,96 @@ extension List: ComponentContainer where Items == ComponentGroup {
     }
 }
 
+// Component used to render an `<picture>` element for responsive image support.
+public struct Picture: Component {
+    /// Type used to define an  Picture source, which points to an image file (or set of them) and associated media queries (and other paramaters).
+    public struct Source {
+        /// image or images  to use - can have parameters
+        public var srcset: String
```

Comment:
Since this is a `Component`, and not a `Node`, I think we should use a bit more verbose naming here. In general, components try to follow Swift's naming conventions as much as possible, while nodes are a direct match with their underlying HTML naming. For example, compare how a link is defined using a node vs a component:

```swift
// Using a node:
.a(.href("https://www.apple.com"), .text("Apple"))

// Using a component:
Link("Apple", url: "https://www.apple.com")
```

So how about naming this `sourceSet`?

Reply:
Done.